### PR TITLE
create hierarchical_layer_en.rst

### PR DESCRIPTION
**Pull request by shadowimp**

### CODE REVIEW

File: doc/v2/howto/rnn/hierarchical_layer_en.rst [hunk 1]
```diff
@@ -1,4 +1,88 @@
 Layers supporting hierarchical sequence as input
 ================================================
 
-TBD
+..	contents::
+
+summary
+=======
+
+The sequence is common data structure in the proccess of NLP(nature language proccess). A single word can be seem as a non-sequence input or a zero-layer input. The sentence make up by words is a one-layer sequence and the senten is a two-layer squence.
```
Comment (abhinavarora) [inline doc/v2/howto/rnn/hierarchical_layer_en.rst]: The sequence is common  -> A sequence is a common

Comment (abhinavarora) [reply doc/v2/howto/rnn/hierarchical_layer_en.rst]: nature language proccess -> **Natural Language Processing**

Comment (abhinavarora) [reply doc/v2/howto/rnn/hierarchical_layer_en.rst]: This line does not make sense -> `A single word can be seem as a non-sequence input or a zero-layer input.`

Comment (abhinavarora) [reply doc/v2/howto/rnn/hierarchical_layer_en.rst]: @shanyi15

Comment (shanyi15) [reply doc/v2/howto/rnn/hierarchical_layer_en.rst]: get, thanks!

**Comment by shanyi15** [comment]
Thank you for your contribution @shadowimp ，however, since the language problem according to @abhinavarora  's comment, please close this PR and we'll open a new one.

File: doc/v2/howto/rnn/hierarchical_layer_en.rst [hunk 2]
```diff
@@ -1,4 +1,92 @@
 Layers supporting hierarchical sequence as input
```
Comment (skylarch) [inline doc/v2/howto/rnn/hierarchical_layer_en.rst]: 所有标题除介词外，单词首字母需大写

**Pull request closed without merging**